modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/macros/APIBuilderUI.macro

```diff
@@ -215,6 +234,22 @@ definition {
 		Click(locator1 = "APIBuilder#EDIT_BUTTON");
 	}
 
+	macro goToEditRelatedEntryInEditApplication {
```

Comment:
The value you defined is {endpointName}, better to update to a proper macro name